docs/source/dataset_card.mdx

```diff
@@ -4,27 +4,25 @@ Each dataset should have a dataset card to promote responsible usage and inform
 This idea was inspired by the Model Cards proposed by [Mitchell, 2018](https://arxiv.org/abs/1810.03993).
 Dataset cards help users understand a dataset's contents, the context for using the dataset, how it was created, and any other considerations a user should be aware of.
 
-This guide shows you how to create a dataset card.
+Creating a dataset card is easy and can be done in a just a few steps:
 
-1. Create a new dataset card by copying this [template](https://raw.githubusercontent.com/huggingface/datasets/main/templates/README.md) to a `README.md` file in your repository.
+1. Go to your dataset repository on the [Hub](https://hf.co/new-dataset) and click on **Create Dataset Card** to create a new `README.md` file in your repository.
 
-2. Generate structured tags to help users discover your dataset on the Hub. Create the tags with the [online Datasets Tagging app](https://huggingface.co/spaces/huggingface/datasets-tagging).
+2. Use the **Metadata UI** to select the tags that describe your dataset. You can add a license, language, pretty_name, the task_categories, size_categories, and any other tags that you think are relevant. These tags help users discover and find your dataset on the Hub.
 
-3. Select the appropriate tags for your dataset from the dropdown menus.
+  <Tip>
 
-4. Copy the YAML tags under **Finalized tag set** and paste the tags at the top of your `README.md` file.
+  For a more complete, but not required, set of tag options you can also use the [Dataset Tagger](https://huggingface.co/spaces/huggingface/datasets-tagging). This'll have a few more tag options like `multilinguality` and `language_creators` which are useful but not absolutely necessary.
```

Comment:
We don't maintain the tagging app anymore, we can redirect to the specs instead:

```suggestion
  For a more complete, but not required, set of tag options you can also look at the [Dataset Card specifications](https://github.com/huggingface/hub-docs/blob/main/datasetcard.md?plain=1). This'll have a few more tag options like `multilinguality` and `language_creators` which are useful but not absolutely necessary.
```

however this feels a bit redundant with the `More YAML tags` section, maybe we can remove it

Reply:
+1 to redundancy! I removed the whole `More YAML tags` section and the link to the Hub docs, which are essentially the same.